### ISSUE TITLE
knapsack_pro 7.6.1 fails with RSpec

### ISSUE THREAD

**Issue by lucasmazza** [KnapsackPro/knapsack_pro-ruby#263]
Dependabot created a PR on our repo to bump `knapsack_pro` from 7.6.0 to 7.6.1 and our build fails inside KnapsackPro internals, probably due a require that was missed on #262. This is the backtrace we get 👇  

```
/bundle/ruby/3.3.0/gems/knapsack_pro-7.6.1/lib/knapsack_pro/formatters/time_tracker.rb:148:in `now': uninitialized constant KnapsackPro::Utils (NameError)
        KnapsackPro::Utils.time_now
                   ^^^^^^^
	from /bundle/ruby/3.3.0/gems/knapsack_pro-7.6.1/lib/knapsack_pro/formatters/time_tracker.rb:23:in `initialize'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/formatters.rb:157:in `new'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/formatters.rb:157:in `add'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/configuration.rb:997:in `add_formatter'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/configuration_options.rb:122:in `block in load_formatters_into'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/configuration_options.rb:122:in `each'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/configuration_options.rb:122:in `load_formatters_into'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/configuration_options.rb:24:in `configure'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/runner.rb:132:in `configure'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/runner.rb:99:in `setup'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/runner.rb:86:in `run'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/runner.rb:71:in `run'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/lib/rspec/core/runner.rb:45:in `invoke'
	from /bundle/ruby/3.3.0/gems/rspec-core-3.13.0/exe/rspec:4:in `<main>'
```

**Comment by ArturT** [comment]
What does the knapsack pro command look like that you use to run tests? I'm wondering why our integration tests did not catch this issue. 

Please could you use 7.6.0 before we fix this issue.

**Comment by lucasmazza** [comment]
Our pipeline runs `KNAPSACK_PRO_LOG_LEVEL=info bin/rake knapsack_pro:rspec`

**Comment by ArturT** [comment]
We've released the knapsack_pro 7.6.2 version that fixes the problem. Please update it.
